charts/customer-center/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: customer-center
+description: Chart for Customer-Center application
+type: application
+version: 0.1.0
+appVersion: 2.0.0-rc.4
+keywords:
+  - customer-center
+home: https://github.com/adfinis-sygroup/customer-center
+sources:
```

Comment:
here you should list the sources of the helm-chart like app-code or docker images and not the chart itself

Reply:
You are right. @vmaillot  

Example: https://github.com/adfinis-sygroup/helm-charts/blob/master/charts/back8sup/Chart.yaml#L9

or

https://github.com/adfinis-sygroup/helm-charts/blob/master/charts/caasperli/Chart.yaml#L8

Reply:
thanks for the hint, changes has been made.

Reply:
LGMT